### ISSUE TITLE
Update wiki re Slack usage expectations

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/whatdotheyknow-theme#1446]
Following the [notes from the latest C&A call](https://docs.google.com/document/d/1E4-O6j2HP42yBVjRGGuTcu8GsAdy8fL1Jog1gShqI4E/edit#heading=h.m5m2jrawrnbj), we should/could update the wiki with the following in relation to Slack:
- Expectations around checking it
- Not overusing the send reply to channel (can be hard to follow, can be relentless)

Comments on the exact wording welcome.

**Comment by garethrees** [comment]
We have some guidance [on the staff wiki](https://wiki.mysociety.org/wiki/Slack) that could be shared here. It won't all be applicable, but just noting for reference when we come to this.

**Comment by HelenWDTK** [comment]
Closed as complete
